translations/base-de.yaml

```diff
@@ -530,25 +530,25 @@ buildings:
         levelShortcut: LVL
     wire:
         default:
-            name: Energy Wire
-            description: Allows you to transport energy.
+            name: Stromkabel
```

Comment:
Das Energieupdate wurde in #425 schon übersetzt

Reply:
Ich glaube `Stromkabel` passt hier besser, als der Begriff `Energiekabel` in der aktuellen Version